src/iocore/net/UnixUDPNet.cc

```diff
@@ -456,6 +455,10 @@ UDPNetProcessorInternal::read_single_message_from_net(UDPNetHandler *nh, UDPConn
       }
 #endif
     }
+    // Use the local address if none of the cmsg to get the destination address is unavailable
```

Comment:
> if none ... is unavailable

That wording seems a little strange; maybe it's a mistake? Otherwise, it might be clearer to say "if any ... is available".

Reply:
I wanted to say "If we couldn't get the address from any cmsgs".